docs/commands/rhoas-kafka-use.adoc

```diff
@@ -1,6 +1,6 @@
-ifdef::env-github,env-browser[:context: cmd]
+docs/commands/-kafka-use.adocifdef::env-github,env-browser[:context: cmd]
```

Comment:
This doesn't look right @wtrocki

Reply:
Yes. Thank you for spotting this. It is still early version. I need to extract all parameters into CLI